src/pages/gallery/courses/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import Head from "next/head";
+import Link from "next/link";
+import { ImageProps } from "next/image";
+import classnames from "classnames";
+
+import { PageBanner } from "../../../components/Banner";
+import { Heading } from "../../../components/Heading";
+import Image from "../../../components/Image";
+
+import styles from "./index.module.scss";
+import cs233Image from "../../../lib/images/cs-233.png";
+
+interface CourseCardProps {
+  image: ImageProps["src"];
+  title: string;
+  href: string;
+  institution: string;
+  rubric: string;
+  ownerName: string;
+  ownerEmail: string;
+  children: React.ReactNode;
+}
+
+const CourseCard: React.FC<CourseCardProps> = ({
+  image,
+  title,
+  href,
+  institution,
+  rubric,
+  ownerName,
+  ownerEmail,
+  children,
+}) => {
+  return (
+    <article className="card">
+      <Link href={href}>
+        <a className="position-relative" style={{ aspectRatio: "4 / 3" }}>
```

Comment:
I picked a random aspect ratio that looked decent; we can play with this as we add more images!

Reply:
I modified this a bit, I think it looks good for now.